### PR TITLE
Add new migrateTo method to the ALCA contract

### PR DESCRIPTION
**Pull request by vtleonardo**
## Scope

Add a new method to allow migrating ALCA to a certain address.

### CODE REVIEW

File: bridge/contracts/AToken.sol [hunk 1]
```diff
@@ -38,15 +38,17 @@ contract AToken is
...
+    function migrate(uint256 amount) public returns (uint256) {
+        return _migrate(msg.sender, amount);
+    }
+
+    /**
+     * Migrates an amount of legacy token (MADToken) to ALCA tokens to a certain address
+     * @param to the address that will receive the alca tokens
+     * @param amount the amount of legacy token to migrate.
+     */
+    function migrateTo(address to, uint256 amount) public returns (uint256) {
```
Comment (z-j-lin) [inline bridge/contracts/AToken.sol:50]: add unit tests

**Review by z-j-lin — CHANGES_REQUESTED** [review]
add test for migrateTo

File: bridge/contracts/AToken.sol (hunk 1 above)
Comment (vtleonardo) [reply bridge/contracts/AToken.sol:50]: Test added!

**Review by Cr0wn-Gh0ul — APPROVED** [review]
LGTM

**Review by z-j-lin — APPROVED** [review]
looks good

**Pull request merged**